config.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/gravwell/gcfg"
+	"github.com/keybase/managed-bots/base"
+)
+
+const version = "1.0.0"
+
+type Options struct {
+	*base.Options
+	um             map[string]string
```

Comment:
Let's give this a more descriptive name for consistency with the other objects in Options.